stdlib/src/commonMain/kotlin/it/unibo/collektive/stdlib/Spreading.kt

```diff
@@ -0,0 +1,118 @@
+package it.unibo.collektive.stdlib
+
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.aggregate.api.operators.share
+import it.unibo.collektive.field.Field
+import it.unibo.collektive.field.operations.minBy
+import kotlin.jvm.JvmName
+import kotlin.jvm.JvmOverloads
+
+/**
+ * Compute the [Distance] from the closest [source], starting from [bottom] and up to [top].
+ *
+ * the [Distance] between neighboring devices is computed using the [metric] function,
+ * the distance summation is governed by the [accumulateDistance] function.
+ */
+inline fun <ID : Any, Distance : Comparable<Distance>> Aggregate<ID>.distanceTo(
```

Comment:
```kotlin
 inline fun <ID : Any, [...] 
```
am I confused with the VMC experiments or did we find that the IDs should be `Comparable`?

Reply:
For `distanceTo`, there's no need to discriminate using `ID`s, so no need to have them comparable. The more generic, the better.